sdk/maps/Azure.ResourceManager.Maps/tests/ScenarioTests/MapsAccountTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Maps.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Maps.Tests
+{
+    public class MapsAccountTests : MapsManagementTestBase
+    {
+        public MapsAccountTests(bool isAsync)
+                    : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        private ResourceGroupResource ResourceGroup { get; set; }
+
+        private MapsAccountCollection MapCollection { get; set; }
+
+        private async Task SetCollection()
+        {
+            ResourceGroup = await CreateResourceGroupAsync();
+            MapCollection = ResourceGroup.GetMapsAccounts();
+        }
```

Comment:
If these properties are not meant to be shared between different test cases, we do not need to make them properties.
We could just let this method to return both the results.

Reply:
Fixed